scala/scalatest/BUILD

```diff
@@ -0,0 +1,10 @@
+package(default_visibility = ["//visibility:public"])
+
+java_import(
```

Comment:
currently java_import doesn't play well enough with our strict-deps mechanism.
We plan on changing this *and* I don't know how many scalatest users use this feature (we use specs2) but maybe it's better to change to `scala_import`